package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "databyss-org-workspace",
-  "version": "1.0.9",
+  "version": "1.1.0",
```

Comment:
The version should be `1.0.10`. We'll use `1.1.x` as the next major release (with the primitives, etc).